labs-ios-starter/Application/View/UI/Custom Objects/CircularButton.swift

```diff
@@ -0,0 +1,100 @@
+// Copyright © 2020 Lambda, Inc. All rights reserved.
+// CircularButton.swift
+
+import UIKit
+
+/// The system icons used throughout the app, enums prevents typos -> UIImage(systemName: "[HERE]")
+enum Icon: String {
+    case plus = "plus"
+}
+
+/// A rounded button with an ICON
+class CircularButton: UIButton {
+    
+    // MARK: - Initializers
+    
+    /// The initializer called directly by the Developer
+    convenience init(withIcon systemName: Icon?, size: CGFloat?) {
+        self.init()
+        
+        if let systemName = systemName?.rawValue {
+            setImage(UIImage(systemName: systemName), for: .normal)
+        }
+        
+        if let size = size {
+            setDimensions(width: size, height: size)
+            layer.cornerRadius = size / 2 // Circular shape            
+        }
+    }
+    
+    /// Programmatic init
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+    }
+    
+    /// Storyboard init
+    required init?(coder: NSCoder) {
```

Comment:
good job adding a storyboard init instead of crashing in case we decide to use a mixed approach